pystac/summaries.py

```diff
@@ -62,6 +64,14 @@ def from_dict(cls, d: Dict[str, Any], typ: Type[T] = Any) -> "RangeSummary[T]":
                                       "fields-normalized.json")
 
 
+class SummaryStrategy(Enum):
+    ARRAY = "v"
+    RANGE = "r"
+    SCHEMA = "s"
+    DONT_SUMMARIZE = False
+    UNDEFINED = True
```

Comment:
```suggestion
    UNDEFINED = True
```

I think "default" or "detect" would be better choices as name. There are rules defined in the fields json readme for the case it's not set, which depends on the data type mostly.

Reply:
changed